From 283946675ac1e225ea35ae463635753c417b2c2d Mon Sep 17 00:00:00 2001
Date: Fri, 13 Jun 2014 19:43:57 +0000
Subject: [PATCH] fai-make-nfsroot: Dereference symlinks when copying apt files

`cp -La $cfdir/apt $NFSROOT/etc` was not working as intended.  The `-a` flag
was overriding the `-L` (dereference symlinks) flag.
---
 bin/fai-make-nfsroot |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/bin/fai-make-nfsroot b/bin/fai-make-nfsroot
index 0fdd4e4..f37b4c0 100755
--- fai-server-4.0.6/bin/fai-make-nfsroot
+++ /usr/sbin/fai-make-nfsroot
@@ -391,7 +391,7 @@ Acquire::Languages none;
 EOF
 
     # this copies sources.list into the nfsroot
-    cp -La $cfdir/apt $NFSROOT/etc
+    cp -aL $cfdir/apt $NFSROOT/etc
 
     add_etc_hosts_entries
 
-- 
1.7.2.5
